commands/cli.go

```diff
@@ -99,8 +104,51 @@ func BuildContextualMenu() []cli.Command {
 			},
 		},
 		{
-			Name:  "decrypt",
-			Usage: "Decrypt an encrypted secret",
+			Name:  "credential",
+			Usage: "Manage cloud credentials",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "cloud",
+					Usage: "cloud, must be one of [GCP]",
+					Value: "GCP",
+				},
+			},
+			Subcommands: []cli.Command{
+				{
+					Name:  "add",
+					Usage: "Add a default credential",
+					Action: func(c *cli.Context) error {
+						var cred credentials.GoogleCredential
+						conf := utils.ReadConfiguration()
+						conf.Init()
+
+						if len(conf.GoogleClient.Data) == 0 {
+							clientData := utils.UserInput("Enter your organizations Sctl google client JSON")
+							conf.GoogleClient = utils.Client{Data: string(clientData)}
+						}
+
+						err := cred.Login(conf, "default")
+						conf.Save()
```

Comment:
I think the config should be saved after the error check, so we are only saving the config if there was no issue logging in